test/qobj.jl

```diff
@@ -143,4 +143,57 @@ end
 @testset "is_hermitian" begin
     @test is_hermitian(sigma_y())
     @test !is_hermitian(sigma_p())
-end
\ No newline at end of file
+end
+
+@testset "change_number_base" begin
+    decimal = -1
+    base = 3
+    num_symbols = 4
+    @test_throws ErrorException Snowflake.change_number_base(decimal, base, num_symbols)
+    
+    decimal = 88
+    @test_throws ErrorException Snowflake.change_number_base(decimal, base, num_symbols)
+
+    decimal = 16
+    symbols = Snowflake.change_number_base(decimal, base, num_symbols)
+    expected_symbols = [0, 1, 2, 1]
+    @test symbols == expected_symbols
+end
+
+@testset "change_to_decimal_integer" begin
+    symbols = [0, 1, 2, 1]
+    base = 3
+    decimal = Snowflake.change_to_decimal_integer(symbols, base)
+    @test decimal == 16
+end
+
+@testset "get_ket_index" begin
```

Comment:
See comment of `change_number_base` test

Reply:
The tests of "private" functions have been removed.